lib/linkingJwtUtils.js

```diff
@@ -3,18 +3,41 @@ const { promisify } = require('util');
 const config = require('./config');
 const logger = require('./logger');
 const findUsersByEmail = require('./findUsersByEmail');
+const fetchCustomDomains = require('./fetchCustomDomains');
+const storage = require('./storage');
+const { qualifyDomain } = require('./urlHelpers');
+
+const fetchRegisteredCustomDomain = async (customDomain) => {
```

Comment:
Does it still make sense to have this helper here? Feels like an odd location with the recent update.

Reply:
I don't love it there either. I don't want to put it with the endpoint because it makes sinon stubbing tougher. I'd like to include it in a module that I can import in to the tests to stub. I mean I can create another file and call it `managementApiRequests.js` or `customDomains.js`.

Reply:
Might make sense to parts of the helper that was added to the put settings route in that file/method as well. I like to separate business logic from routing/request/response logic.

Reply:
moved to `settingsUtils.js`